chore: add docs for feature flag quota limiting (#10653)
PostHog/posthog.com: contents/docs/integrate/feature-flags-code/_snippets/feature-flags-code-api.mdx

> 
> This enables partial updates to currently active flags in your clients.

> #### Quota Limiting

Reviewer: ```suggestion
#### Quota limiting
```

> If your organization exceeds its feature flag quota, the `/decide` endpoint will return a modified response:

Reviewer: Just clarifying when we are looking at here as most people won't be familar with what the decide response looks like :)
```suggestion
If your organization exceeds its feature flag quota, the `/decide` endpoint will return a modified response with `quotaLimited`:
```

```json
{
  "config": {
    "enable_collect_everything": true
  },
  "toolbarParams": {},
  "isAuthenticated": false,
  "supportedCompression": [
    "gzip",
    "lz64"
  ],
  "featureFlags": {},
  "errorsWhileComputingFlags": false,
  "featureFlagPayloads": {},
  "quotaLimited": ["feature_flags"]
  // ... other fields, not relevant to feature flags
}
```

When you receive a response with `quotaLimited` containing `"feature_flags"`, it means:
1. Your feature flag evaluations have been temporarily paused because you've exceeded your feature flag quota

> 2. If you want to continue evaluating feature flags, you can increase your quota in the PostHog UI or contact support

Reviewer: ```suggestion
3. If you want to continue evaluating feature flags, you can increase your quota in [your billing settings](https://us.posthog.com/organization/billing) under **Feature flags & Experiments** or [contact support](https://us.posthog.com/#panel=support%3Asupport%3Abilling%3A%3Atrue)
```
